tests/unit/moduleapi/getkeys.tcl

```diff
@@ -86,4 +86,43 @@ start_server {tags {"modules"}} {
     test "Unload the module - getkeys" {
         assert_equal {OK} [r module unload getkeys]
     }
+
+    test {KEYS command does not return expired keys when allow_access_expired is 0} {
```

Comment:
the default value of allow_access_expired is 0, this code is bound to be covered by other tests, so i think we don't need this test.